tests/db_functions/test_sin.py

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
chop blank line (check other files for this issue)

Reply:
Thank you. I chopped the blank lines and updated releases/2.1.txt. The new commit still cannot trigger a build.

Reply:
I saw an error in the Jenkins log and tried the steps here: https://stackoverflow.com/questions/36336188/jenkins2-branch-indexing-java-lang-illegalargumentexception-bytestring-null.